ext/opcache/zend_accelerator_debug.c

```diff
@@ -37,7 +37,7 @@ void zend_accel_error(int type, const char *format, ...)
 
 	if (type <= ZCG(accel_directives).log_verbosity_level) {
 
-	timestamp = time(NULL);
+	timestamp = php_time(NULL);
```

Comment:
```
/home/travis/build/php/php-src/ext/opcache/ZendAccelerator.c: In function ‘zend_accel_init_shm’:
/home/travis/build/php/php-src/ext/opcache/ZendAccelerator.c:2568:39: error: implicit declaration of function ‘php_time’ [-Werror=implicit-function-declaration]
  ZCSG(start_time) = zend_accel_get_time();
                                       ^
```

I think you should skip this one, as I'm not sure if it's wise to include the ext/date header for this